lib/data/wrapper.js

```diff
@@ -182,6 +183,10 @@ const data = {
         if (response) {
             objGetInfo.response = response;
         }
+        if (objGetInfo.key === null) {
```

Comment:
do you have to set the key to null?  Would it be possible to set it to something more descriptive? Very legacy deployments would have objectGetInfo.key = undefined which could be confusing.